web/modules/custom/collection/collection.info.yml

```diff
@@ -11,19 +11,3 @@ dependencies:
   - options
   - rdf_entity
   - text
-
-config_devel:
-  - core.entity_form_display.collection.collection.default
-  - core.entity_view_display.collection.collection.default
-  - field.field.collection.collection.field_collection_closed
-  - field.field.collection.collection.field_collection_create_elibrary
-  - field.field.collection.collection.field_collection_logo
-  - field.field.collection.collection.field_collection_metadata_url
-  - field.field.collection.collection.field_collection_pre_moderation
-  - field.field.collection.collection.field_collection_schedule
-  - field.storage.collection.field_collection_closed
-  - field.storage.collection.field_collection_create_elibrary
-  - field.storage.collection.field_collection_logo
-  - field.storage.collection.field_collection_metadata_url
-  - field.storage.collection.field_collection_pre_moderation
-  - field.storage.collection.field_collection_schedule
```

Comment:
It seems like you have to be introduced to the awesomeness of the `config_devel` module. I'll show you a demo soon, you'll love it :)